src/snapshot.ts

```diff
@@ -207,8 +144,40 @@ export class Snapshot {
   static formatName_(projectId: string, name: string) {
     return 'projects/' + projectId + '/snapshots/' + name.split('/').pop();
   }
+
+
+  create(gaxOpts?: CallOptions): Promise<CreateSnapshotResponse>;
+  create(callback: CreateSnapshotCallback): void;
+  create(gaxOpts: CallOptions, callback: CreateSnapshotCallback): void;
+  create(
+      gaxOpts?: CallOptions|CreateSnapshotCallback,
+      callback?: CreateSnapshotCallback): void|Promise<CreateSnapshotResponse> {
+    if (!(this.parent instanceof Subscription)) {
+      throw new Error(`Subscription#snapshot`);
```

Comment:
Sorry for any miscommunication on my part, but what I meant when I said drop the JSDoc part was just removing the `{@link}` tag. We should keep the beginning part that said `This is only available if you accessed this object through`.

Reply:
Done